public/app/features/alerting/unified/components/receivers/form/ChannelSubForm.tsx

```diff
@@ -73,22 +73,14 @@ export function ChannelSubForm<R extends ChannelValues>({
   useEffect(() => {
     // Restore values when switching back from a changed integration to the default one
     const subscription = watch((v, { name, type }) => {
-      const value = name ? v[name] : '';
+      const value = name ? getValues(name) : '';
```

Comment:
I think the only change we want to to is to use Lodash's `get` function to get the correct property value by the `name` variable
```suggestion
      const value = name ? get(v, name) : '';
```
`getValues` should also work, but I'm not sure if it always contain the most up to date from values. The `v` should include the newest changes

Reply:
Looks like the bug was introduced when upgrading `react-hook-form` version